contributors/design-proposals/standardization-of-development-process.md

```diff
@@ -0,0 +1,282 @@
+# Proposal to Standardize Development Process
+
+## Links
+```
+---
+Links:
+- Issues:
+  - [someIssueURL]()
+- PRs:
+  - https://github.com/kubernetes/community/pull/967
+- MailingListDiscussions:
+  - https://groups.google.com/forum/#!topic/kubernetes-dev/65A-3ULYPB0
+- Documentation:
+  - [someDocsLinkURL]()
+```
+
+## Responsible SIG(s)
+
+- SIG Release
+- SIG PM
+- SIG Architecture
+- SIG Governance
+- Technical Steering Committee
+
+## Summary
+
+A standardized development process for Kubernetes is proposed in order to:
+- provide a common structure for proposing changes to Kubernetes
+- ensure that the motivation for a change is clear
+- allow for the enumeration stability milestones and stability graduation
+  criteria
+- persist project information in a Version Control System (VCS) for future
+  Kubernauts
+- support the creation of _high value user facing_ information such as:
+  - release notes
+  - release announcement blog
+  - a cross project development road map
+- support development across multiple repositories beyond `kubernetes/kubernetes`
+- begin to move away from using GitHub issues for feature requests and reserve them
+  for tracking work in flight
+
+## Motivation
+
+Today it is becoming fairly difficult to track changes being made across all
+Special Interest Groups (SIGs) within Kubernetes. For cross project SIGs such
```

Comment:
I might qualify this statement and say it's difficult to know the chain of custody of any change in Kubernetes.  That chain currently includes all program-level SIGs, sponsor SIGs, reviewers, approvers, automation, and eventually a release team.  In the scope of this proposal, you add conceptual stewardship on top of it all.  It's critical that even a one-line change has a defined lifecycle.  We also need visibility of all the custodial handoffs so we know when things stall, or reach a bottleneck in the value stream.  That will allow traceability, and the ability to focus attention on areas of the RFC and release process that need it the most.  (My magic 8 Ball says architectural review will be an early bottleneck)

Reply:
The "one line change" thing is scaring me, to be honest.  There will be a significant number of bug fixes and enhancements that are purely contained within a single SIG and shouldn't have to activate this process.

Reply:
I think that's up to the SIG to decide as part of their chartering document. I do not agree that there will be a significant number of enhancements that fall within a single SIG that would not benefit from a more formalized process that will allow SIG Docs, SIG PM to communicate the importance of those enhancements beyond the SIG. I'm sure @radhikapc or @Bradamant3can can explain the challenge of  communicating what went into the release and why it's important far better than I can.